symfony.lock

```diff
@@ -231,6 +231,12 @@
     "imagine/imagine": {
         "version": "1.2.4"
     },
+    "knplabs/knp-components": {
+        "version": "v3.1.0"
+    },
+    "knplabs/knp-paginator-bundle": {
```

Comment:
knp-paginator-bundle has been removed.
It was used with Articles, but that functionality has been removed.